build.gradle.kts

```diff
@@ -13,6 +13,7 @@ import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 buildscript {
     repositories {
         maven { setUrl("https://jitpack.io") }
+        maven { setUrl("http://dl.bintray.com/jetbrains/intellij-plugin-service") }
```

Comment:
Is it possible to use https for this repository?

Reply:
fixed it